files/galaxy/tpv/destinations.yml

```diff
@@ -101,9 +101,15 @@ destinations:
       submit_request_cpus: "{cores}"
       submit_request_memory: "{mem}"
       docker_volumes: "$default,$job_directory:rw,$tool_directory:ro,/cvmfs/data.galaxyproject.org:ro"
+      singularity_volumes: "$default,$job_directory:rw,$tool_directory:ro,/cvmfs/data.galaxyproject.org:ro"
```

Comment:
so `$default` contains all the `data` mount points? If not here by setting `singularity_volumes` we overwrite the default from the `basic_singularity_destination` destination which exclusively includes all those `data` mount points.

Reply:
This was the goal, yes. They are not available on Pulsar nodes

Reply:
Right! Sorry, overlooked it.

Reply:
I'm not sure why the condor scratch dir is not needed on pulsar ... maybe because of htcondor 10?
Just keep an eye on this variable and strange errors from condor.